api/slotify_group_impl.go

```
@@ -20,8 +20,10 @@ func (s Server) DeleteSlotifyGroupsSlotifyGroupIDLeaveMe(w http.ResponseWriter,
 	defer cancel()
 
 	userID, ok := r.Context().Value(UserIDCtxKey{}).(uint32)
+	reqUUID, _ := ReadReqUUID(r)
```

you are ignoring the error, this is like having a try-catch but doing nothing in the catch. you need to do something like

```go
var reqUUID string
if reqUUID, err = ReadReqUUID(r); err != nil{
//error was not nil, do something
}


```

---

Also do this after the `!ok` check of userID

---

Also get rid of the `ReadReqUUID` function, just do it inline like 
```go
userID, ok := r.Context().Value(UserIDCtxKey{}).(uint32
```
you cant make a function simpler than that